src/shared/components/VideoTileBase/VideoTileBase.styles.tsx

```diff
@@ -85,6 +85,15 @@ export const Container = styled.article<LoadingProps>`
   width: 100%;
   display: inline-flex;
   flex-direction: column;
+  min-height: 250px;
+
+  ${media.sm} {
+    min-height: 260px;
+  }
+
+  ${media.xxl} {
+    min-height: 270px;
+  }
```

Comment:
Is the video tile height really the problem here? I would expect that this is more of an layout/alignment issue with the big number.

Reply:
Unfortunately it uses `align-items: center;`, problem appears when loaded video has more lines than skeleton. I will try figure out some other way to position it.